github/resource_github_repository_file.go

```diff
@@ -10,6 +11,7 @@ import (
 
 	"github.com/google/go-github/v57/github"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"golang.org/x/crypto/openpgp"
```

Comment:
https://github.com/golang/go/issues/44226

Need to replace this library, and add some tests for the actually using the sign commit opt.